### PR TITLE
giga slop pr mk7

### PR DESCRIPTION
**Pull request by JTGSZ**
### IF TESTMERGING BE ON TO MONITOR THINGS
number 7 (so its lucky this time)

rewrites a ton of the triumph shit
adds triumph buy menu
merges pilgrim and adventurers into their complete hobo form
gives them a brand new selection menu w extra options
retypes beggar typepaths to beggar typepaths as im now using vagrants as a name
swaps category bools to bitflags on classes
fixes the fullscreen overlays being fucked on layers for some reason (idk happened recently)

### CODE REVIEW

**Comment by Knightfall5** [comment]
https://cdn.discordapp.com/attachments/1225593965643042817/1240312797745647668/image.png?ex=66461ace&is=6644c94e&hm=e0a295efe60186014af0f868aba731247a2233e4b0034400f3e4d92d486fddc7&

https://cdn.discordapp.com/attachments/1237071216767012966/1240315143720669365/image.png?ex=66461cfe&is=6644cb7e&hm=73e5c7473b5d5de4225e03129a1d950f9d1eec51e4e6709ee4bd8419a75ff592&

RNG classes is lame and stupid, people have been saying this since the server first launched. Also shit is extremely broken, both balance wise and code wise.

**Comment by TheManWithNoHands** [comment]
They where not on to monitor things

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/datum/triumph_buy/wipe_triumphs
+	triumph_buy_id = "WipeTriumphs"
+	desc = "Burn down the hall of triumphs! Bring everyone back to ZERO!"
+	triumph_cost = 100
```
Comment (Real-MAGNUM) [inline code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm:4]:
```suggestion
	triumph_cost = 500
```

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/datum/triumph_buy/grenzelhoft_maximum
+	triumph_buy_id = "Grenzelhoftmaxx"
+	desc = "Everyone is a human from Grenzelhoft!"
+	triumph_cost = 30
```
Comment (Real-MAGNUM) [inline code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm:4]:
```suggestion
	triumph_cost = 50
```

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/character/pick_any_class.dm [hunk 3]
```diff
@@ -0,0 +1,59 @@
+/datum/triumph_buy/pick_any_class
+	triumph_buy_id = "PickAny"
+	desc = "Get single run of a class that can pick any class on the vagrant class selection! WARNING: MAY BE BUGGY"
+	triumph_cost = 10
```
Comment (Real-MAGNUM) [inline code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/character/pick_any_class.dm:4]:
```suggestion
	triumph_cost = 2
```

**Review by Real-MAGNUM — CHANGES_REQUESTED** [review]
Change these prices, and for the love of christ please change the text colors here to a unified set. Ask @Takyon69 for a pallete. 
image

untestmerging

**Comment by K4rlox** [comment]
adventurer class pick thing does not scale with PQ, it used to do so
i really think it should still do so

**Comment by Real-MAGNUM** [comment]
> adventurer class pick thing does not scale with PQ, it used to do so i really think it should still do so

There was supposed to be a slot-purchasing functionality that permanently (per character) increased the amount of slots purchased. However, the price is 1 and it doesn't seem to even work.

**Comment by Real-MAGNUM** [comment]
Stop making fucking meme PR names or I'll start closing them every time regardless of changes.

**Comment by JTGSZ** [comment]
Closing this PR as I'm going to redo a lot of stuff based on feedback/new design

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm (hunk 1 above)
Comment (CVSthePharmacy) [reply code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm:4]: GO FUCK YOURSELF NO, 500 WOULD TAKE AN ACTUAL 5 FUCKING MONTHS, USE YOUR FUCKING HEAD

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm (hunk 2 above)
Comment (CVSthePharmacy) [reply code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm:4]: WAY too much for a funny gimmick

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm (hunk 1 above)
Comment (assortedbeads) [reply code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm:4]: Good, erasing peoples progress shouldnt be easy

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm (hunk 2 above)
Comment (assortedbeads) [reply code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/round_effects/grenzelhoft.dm:4]: Races have stats, its not a funny gimmick its a straight up mechanical effect
Hell it should cost more

File: code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm (hunk 1 above)
Comment (CVSthePharmacy) [reply code/controllers/subsystem/rogue/triumphs/triumph_buy_datums/misc/wipe_triumphs.dm:4]: > Good, erasing peoples progress shouldnt be easy

hey cocksucker, that's 5-6 months of work, i'd at most be ok with 150 or 175

**Pull request closed without merging**